_index.md

```diff
@@ -1,118 +1,124 @@
 ---
 title: TiDB Introduction
-summary: Learn about the NewSQL database TiDB that supports HTAP workloads.
-aliases: ['/docs/dev/','/docs/dev/adopters/','/tidb/dev/adopters']
+aliases: ["/docs/dev/", "/docs/dev/adopters/", "/tidb/dev/adopters"]
+hide_sidebar: true
+hide_commit: true
 ---
 
-# TiDB Introduction
+<LearningPathContainer platform="tidb" title="TiDB" subTitle="TiDB is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. Find the guide, samples, and references you need to use TiDB.">
 
-[TiDB](https://github.com/pingcap/tidb) (/’taɪdiːbi:/, "Ti" stands for Titanium) is an open-source, distributed, NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability. TiDB can be deployed on-premise or in-cloud.
+<LearningPath label="Learn" icon="cloud1">
 
-Designed for the cloud, TiDB provides flexible scalability, reliability and security on the cloud platform. Users can elastically scale TiDB to meet the requirements of their changing workloads. [TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview) helps manage TiDB on Kubernetes and automates operating tasks, which makes TiDB easier to deploy on any cloud that provides managed Kubernetes. [TiDB Cloud](https://pingcap.com/tidb-cloud/), the fully-managed TiDB service, is the easiest, most economical, and most resilient way to unlock the full power of [TiDB in the cloud](https://docs.pingcap.com/tidbcloud/), allowing you to deploy and run TiDB clusters with just a few clicks.
+[What is TiDB](https://docs.pingcap.com/tidb/stable/overview)
```

Comment:
This PR will be merged to the master branch, while all links are "stable" ones. The versions are not consistent.

Reply:
Since the "stable" links are what are displayed by default when users access the docs, we'll use "stable" links in the files, but will update the version info in links for other TiDB versions.